test/Common/HttpApiHandlerTests.cs

```diff
@@ -474,7 +474,7 @@ public async Task HandleGetStatusRequestAsync_Failed_Orchestration_Config_Respon
             var instanceId = Guid.NewGuid().ToString();
             var clientMock = new Mock<IDurableClient>();
             clientMock
-                .Setup(x => x.GetStatusAsync(instanceId, false, false, true))
+                .Setup(x => x.GetStatusAsync(instanceId, false, false, true, null))
                 .Returns(Task.FromResult(list.First()));
             var httpApiHandler = new ExtendedHttpApiHandler(clientMock.Object);
 

```

Comment:
Can we get a test that sets the `eventTypes` argument to a non-null value? It would be good to test it :)

Reply:
nevermind, I see this overlaps with the E2E test. no need for a unit test then